katalog/velero/velero-prem/minio/kustomization.yaml

```diff
@@ -0,0 +1,54 @@
+namespace: kube-system
+
+resources:
+  - rbac.yaml
+  - init-job.yaml
+  - service.yaml
+  - sts.yaml
+
+vars:
+  - name: MINIO_NAMESPACE
+    objref:
+      kind: Service
+      name: minio
+      apiVersion: v1
+    fieldref:
+      fieldpath: metadata.namespace
+  - name: MINIO_SERVICE_NAME
+    objref:
+      kind: Service
+      name: minio
+      apiVersion: v1
+    fieldref:
+      fieldpath: metadata.name
+  - name: MINIO_STS_NAME
+    objref:
+      kind: StatefulSet
+      name: minio
+      apiVersion: apps/v1
+    fieldref:
+      fieldpath: metadata.name
+  - name: MINIO_SETUP_JOB_NAME
+    objref:
+      kind: Job
+      name: minio-setup
+      apiVersion: batch/v1
+    fieldref:
+      fieldpath: metadata.name
+
+images:
+  - name: minio/minio
+    newTag: RELEASE.2020-01-25T02-50-51Z
+  - name: minio/mc
+    newTag: RELEASE.2020-01-25T03-02-19Z
+
+generatorOptions:
```

Comment:
You shouldn't need `disableNameSuffixHash: true` as `kustomize` will take care of updating the secret name in this case.

Reply:
Discussed here: https://github.com/sighupio/fury-kubernetes-dr/pull/10#discussion_r372801628 (i bet)

Reply:
Ok, but in the on-prem case it the secret would be generated just fine and the name's references will be updated accordinlgy. Therefore, I see no issue in keeping `secretGenerator`'s default behaviour but maybe I am just missing the big picture.

Reply:
`cloud-credentials` secret in the aws,gke,aks deployment is created outside this repository. this is why we need an static secret name.